js/package.json

```diff
@@ -37,11 +37,11 @@
     "gulp-wrap": "^0.15.0",
     "istanbul": "^0.4.5",
     "jasmine-core": "^3.5.0",
-    "jshint": "^2.11.0",
+    "jshint": "^2.9.5",
     "jshint-stylish": "^2.2.1",
     "karma": "^4.4.1",
     "karma-coverage": "^1.1.2",
-    "karma-firefox-launcher": "^1.3.0",
+    "karma-firefox-launcher": "^1.2.0",
```

Comment:
why is the version going backwards?

Reply:
is this file supposed to be deleted?
there is a new and correct one in the root folder